boltdesignsystem/bolt
# DS-204: Print Styles

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-204](https://pegadigitalit.atlassian.net/browse/DS-204)

## Summary

Add a print style utility for academy to a demo. 

## Details

- `e-bolt-printable` class on the `<body>` of a page will hide everything in print mode.
- `e-bolt-printable__article` class on the article container will display the container in print mode.
- Added `all` to the style lint config. It has to come first, otherwise it will override everything if you do `all: initial` after all the rules you’ve written

## How to test

Open the [demo page](https://feature-ds-204-print-styles.boltdesignsystem.com/pattern-lab/patterns/50-pages-60-academy-05-pages-t2-inner-pages-topic-details-acd-topic-detail-page/50-pages-60-academy-05-pages-t2-inner-pages-topic-details-acd-topic-detail-page.html) we added the element classes to and use the browser's print utility to see a preview of the print styles. Either print or save as PDF, only the article would display in print mode.
